modules/apps/object/object-rest-api/src/main/java/com/liferay/object/rest/dto/v1_0/ObjectEntry.java

```diff
@@ -397,7 +397,33 @@ public void setKeywords(
 	@JsonAnyGetter
 	@Schema
 	@Valid
+	public Map<String, UnsafeSupplier<Object, Exception>> getLazyProperties() {
+		return _lazyProperties;
+	}
+
 	public Map<String, Object> getProperties() {
+		if (properties != null) {
+			return properties;
```

Comment:
What if someone has set the `properties` map and the `_lazyProperties` too? We would be ignoring the `_lazyProperties` ones